src/cms/page-builder-media-slider.md

```diff
@@ -118,6 +118,36 @@ You can save time by duplicating a slide that has already been configured with t
 
 1. When complete, click <span class="btn">Save</span> in the upper-right corner to close the _Edit Slide_ page.
 
+### Add widget on a slide
+
+You can add any [widgets types]({% link cms/widget-types.md %}) to your slide in a page bilder stage with the following steps:
+
+1. [Create the widget] ({% link cms/widget-create.md %}) you want to see on a slide.
+
+1. Open the page, block, or dynamic block where you want to place the slider and expand the **Content** section.
+
+1. In the Page Builder panel, expand **Media** and drag a **Slider** placeholder to a row, column, or tab on the stage.
+
+1. Click in the slider container to display the [text editor]({% link cms/editor.md %}).
+
+1. In the text editor click the **Insert Widget** ![]({% link images/images-ee/insert-widget-icon.png  %})).
```

Comment:
I would combine these to a single step and the widget icon already exists, so let's remove the new image file.

```suggestion
1. Click in the slider container to display the [text editor]({% link cms/editor.md %}) toolbar and click the _Insert Widget_ ![]({% link images/images/editor4-toolbar/btn-insert-widget.png  %})) icon.
```

Reply:
Done ✅